src/common/OfflineNotice.js

```diff
@@ -37,17 +39,14 @@ type Props = {|
 class OfflineNotice extends PureComponent<Props> {
   render() {
     const { isOnline } = this.props;
+    if (isOnline) {
+      return <View key={key} style={styles.none} />;
+    }
+
     return (
-      <AnimatedComponent
-        stylePropertyName="height"
-        fullValue={30}
-        useNativeDriver={false}
-        visible={!isOnline}
-        style={styles.block}
-        delay={300}
-      >
-        {!isOnline && <Label style={styles.text} text="No Internet connection" />}
-      </AnimatedComponent>
```

Comment:
No objection to this bit really -- but I am curious about that symptom! The banner would get "stuck in mid-animation until a refresh occurs" -- does that mean it'd be stuck indefinitely (like multiple seconds) until you did something to unstick it?

Reply:
Exactly so. (Scrolling the list was sufficient.)

Reply:
> Exactly so. (Scrolling the list was sufficient.)

Huh! Fascinating bug. Like it was just missing a timer to tell it to actually keep drawing new frames, or something.